docs/current_docs/integrations/snippets/gitlab-kubernetes.yml

```diff
@@ -0,0 +1,17 @@
+.dagger:
+  image: alpine:latest
+  variables:
+    _EXPERIMENTAL_DAGGER_RUNNER_HOST: "unix:///var/run/dagger/buildkitd.sock"
+    DAGGER_CLOUD_TOKEN: $DAGGER_CLOUD_TOKEN
+  before_script:
+    - apk add curl
+    - curl -L https://dl.dagger.io/dagger/install.sh | BIN_DIR=$HOME/.local/bin sh
```

Comment:
If possible, this should be already provisioned in the runner before the job starts. We are doing something similar in our latest GitHub Actions integration. @matipan shares that context.

This will be increasingly important for the next 3-6 months as the CLI & Engine architecture is getting overhauled. FTR:
- https://github.com/dagger/dagger/issues/5583
- https://github.com/dagger/dagger/issues/5484